### PR TITLE
FM2-73 Remove the OpenMRS Services from Translators

### PR DESCRIPTION
**Pull request by VaishSiddharth**
## Issue I worked on

see https://issues.openmrs.org/browse/FM2-73

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**.

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by ibacher** [comment]
@VaishSiddharth This looks good to me and seems to be passing the tests. Do remember to squash your commits together (this is mostly for practice working in the OpenMRS community; this repo is setup to squash commits on merge).

**Comment by VaishSiddharth** [comment]
@ibacher Sure I'll squash the commits in future.

File: api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java [hunk 1]
```diff
@@ -89,4 +91,13 @@ public PatientIdentifierType getPatientIdentifierTypeByNameOrUuid(String name, S
 		return sessionFactory.getCurrentSession().createCriteria(Patient.class).createAlias("names", "names")
 		        .add(ilike("names.familyName", family, MatchMode.START)).list();
 	}
+	
+	@Override
+	public List<PersonAttribute> getActiveAttributesByPatientAndAttributeTypeUuid(Patient patient,
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java]: Technically, we don't need this function at all. We can just use the implementation in the `PersonDao`, which simplifies maintenance.

Comment (VaishSiddharth) [reply api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java]: @ibacher I am removing that code

**Pull request merged**